src/templates/cart.liquid

```diff
@@ -102,6 +102,19 @@
   </form>
 {% else %}
   <h1>{{ 'cart.general.title' | t }}</h1>
-  <p>{{ 'cart.general.empty' | t }}</p>
-  <p>{{ 'cart.general.continue_browsing_html' | t }}</p>
+
+  {% comment %}
+  Cart Empty State
+  {% endcomment %}
+  <div class="cookies">
+    <p>{{ 'cart.general.empty' | t }}</p>
+    <p>{{ 'cart.general.continue_browsing_html' | t }}</p>
+  </div>
+
+  {% comment %}
+  Cart no cookies state
```

Comment:
Please put a small blurb in here that explains browser cookies are required to build a cart